cgi-bin/DW/Media.pm

```diff
@@ -110,9 +110,15 @@ sub upload_media {
 
     # set the security
     my $sec = $opts{security} || 'public';
+    $sec = 'usemask' if $sec =~ /^(?:friends|access)$/;
```

Comment:
Okay. I was trying to make it easier.

I won't accept this patch as-is. It needs to be explicit. If you feel that means you have to go change every other occurrence, that's your choice. It doesn't seem like a good idea to block this improvement on those fixes, but that's your clal.